Client/src/components/SiteSearch/SiteSearch.tsx

```diff
@@ -7,7 +7,7 @@ import { WdkService } from '@veupathdb/wdk-client/lib/Core';
 import { WdkDependenciesContext } from '@veupathdb/wdk-client/lib/Hooks/WdkDependenciesEffect';
 import { useWdkService } from '@veupathdb/wdk-client/lib/Hooks/WdkServiceHook';
 import { DEFAULT_STRATEGY_NAME } from '@veupathdb/wdk-client/lib/StoreModules/QuestionStoreModule';
-import { makeClassNameHelper } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
+import { makeClassNameHelper, safeHtml } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
```

Comment:
Do you need `safeHtml`?

```suggestion
import { makeClassNameHelper } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
```

Reply:
Ah, yes. I will remove it. Thanks!